### PR TITLE
Adding readme for NotificationHubs

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

**Comment by pragnagopa** [comment]
Fixes #64

File: README.md [hunk 1]
```diff
@@ -276,6 +276,36 @@ public static void DocumentClient(
...
+
+Azure Notification Hub must be configured for the Platform Notifications Services (PNS) you want to use. For more information on configuring an Azure Notification Hub and developing a client applications that register for notifications, see [Getting started with Notification Hubs] (https://azure.microsoft.com/en-us/documentation/articles/notification-hubs-windows-store-dotnet-get-started/) and click your target client platform at the top.
+
+The following sample sends windows toast notification on a TimerTrigger
+
+```csharp
+ public static void SendWindowsNotification(
+    [TimerTrigger("*/30 * * * * *")] TimerInfo timerInfo,
```
Comment (mathewc) [inline README.md]: On the main readme page it would be best if you had a real world example. Nobody is going to set up a timer to send notifications :)

**Comment by pragnagopa** [comment]
@mathewc / @brettsam sign off?

**Comment by mathewc** [comment]
Sure. Please squash and merge manually (not via Green button :))

**Pull request merged**